### PR TITLE
Prevent git errors from showing in command output

### PR DESCRIPTION
**Pull request by dacook**
#### What? Why?
This would be a very rare case, but one time the git command failed, printing out an obscure "fatal" message to stderr which caused confusion.

https://openfoodnetwork.slack.com/archives/C02G54U7H/p1690478240843989?thread_ts=1690416052.303299&cid=C02G54U7H

> ```
> DLE9R1G:~/openfoodnetwork$ bundle exec rake db:test:prepare
> fatal: No names found, cannot describe anything.
> ```

#### What should a dev test?
I'm not sure it's worth testing, but if you want to: 

1. Break git somehow, eg `mv .git .git.bak`
2. Run any rails process, eg `rails s` or `rake db:test:prepare`
3. No "fatal" errors should appear in the first line of output. The rest of the process should succeed as usual
4. Don't forget to fix your git repo again!

Screen Shot 2023-07-28 at 12 14 41 pm
The version number doesn't appear, but there are no errors.
Screen Shot 2023-07-28 at 12 13 53 pm

I tested this already.

#### Release notes

Changelog Category: Technical changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: lib/git_utils.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class GitUtils
```
Comment (jibees) [inline lib/git_utils.rb:1]:
```suggestion
+ # frozen_string_literal: true
+ 
class GitUtils
```

**Review by jibees — APPROVED** [review]
👌

**Review by rioug — APPROVED** [review]
Nice !

**Review by mkllnk — APPROVED** [review]

File: lib/git_utils.rb (hunk 1 above)
Comment (dacook) [reply lib/git_utils.rb:3]: 🙌  Thankyou!

**Comment by dacook** [comment]
I don't believe further testing is required and no-one else mentioned it so I will merge.
Bypassing the approval rule, because the last commit is a comment change only.

**Pull request merged**